commands/store.go

```diff
@@ -33,14 +33,16 @@ const (
 func init() {
 	// Setup flags that are used by multiple commands (variables defined in faas.go)
 	storeCmd.Flags().StringVarP(&gateway, "gateway", "g", defaultGateway, "Gateway URL starting with http(s)://")
+	storeCmd.PersistentFlags().StringVarP(&storeAddress, "url", "u", defaultStore, "Alternative URL starting with http(s)://")
```

Comment:
Since 3 sub commands need this flag, better to make it persistent to the namespace `store`